src/daemon/rpmostreed-transaction.cxx

```diff
@@ -313,6 +313,99 @@ transaction_gpg_verify_result_cb (OstreeRepo *repo, const char *checksum,
                                                  g_strdup (checksum));
 }
 
+static void
+transaction_output_cb (RpmOstreeOutputType type, void *data, void *opaque)
```

Comment:
This function mostly just *moved* from `sysroot_output_cb`, with just some tweaks in the first section because we no longer need to check for a transaction being present, because we now only call this when we have a running one.